analyzers/tests/SonarAnalyzer.UnitTest/Rules/DatabasePasswordsShouldBeSecureTest.cs

```diff
@@ -57,6 +61,35 @@ public class DatabasePasswordsShouldBeSecureTest
                                     ParseOptionsHelper.FromCSharp8,
                                     GetReferences("3.1.11", "3.19.80"));
 
+        [DataTestMethod]
+        [DataRow(@"TestCases\WebConfig\DatabasePasswordsShouldBeSecure\Values")]
+        [DataRow(@"TestCases\WebConfig\DatabasePasswordsShouldBeSecure\UnexpectedContent")]
+        [TestCategory("Rule")]
+        public void DatabasePasswordsShouldBeSecure_CS_WebConfig(string root)
+        {
+            var webConfigPath = Path.Combine(root, WebConfig);
```

Comment:
I would extract a method `GetWebConfigPath` in order to enable reuse (I can see at least 3 possible calls). It will be useful if we later want to change the way the path is computed.